### PR TITLE
Properly import "Crimson Text" and "Source Sans"

### PR DESCRIPTION
**Pull request by caesar2164**
Also only import the few useful weights to make page loads faster

@stvstnfrd here's a quick fix to clean up our font imports.

FYI @mreyk. Thanks for the heads up. This cleans up the imports and makes them all work properly.

### CODE REVIEW

File: templates/theme-head-extra.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <meta name="viewport" content="width=device-width, target-densitydpi=device-dpi">
     
 <link rel="stylesheet" href="${static.url('themes/lagunita/css/vendor/su-identity.css')}" />
-<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,900,300italic,400italic,700italic,900italic,' rel='stylesheet' type='text/css'>
```
Comment (stvstnfrd) [inline templates/theme-head-extra.html]: Why remove the `type` attribute?

Comment (caesar2164) [inline templates/theme-head-extra.html]: whoops, didn't mean to.

**Comment by caesar2164** [comment]
@stvstnfrd

File: templates/theme-head-extra.html (hunk 1 above)
Comment (caesar2164) [inline templates/theme-head-extra.html]: not removed now

**Comment by caesar2164** [comment]
[Properly import "Crimson Text" : Fix font imports in lagunita theme](https://trello.com/c/BhnW1lct/928-properly-import-crimson-text-fix-font-imports-in-lagunita-theme)

**Pull request merged**